_config_base.yml

```diff
@@ -89,12 +89,12 @@ release_info:
     name: v21.2.4
     start_time: 2022-01-10 11:01:26.34274101 +0000 UTC
     version: v21.2.4
-  # v22.1:
-  #   build_time: 2021/11/01 11:00:26 (go1.16.6)
-  #   docker_image: cockroachdb/cockroach-unstable
-  #   name: v21.2.0
-  #   start_time: 2021-11-01 11:01:26.34274101 +0000 UTC
-  #   version: v21.2.0
+  v22.1:
+    build_time: 2022/18/01 11:00:26 (go1.16.6)
+    docker_image: cockroachdb/cockroach-unstable
+    name: v22.1.0-alpha.1
+    start_time: 2022-18-01 11:01:26.34274101 +0000 UTC
```

Comment:
Oooops just saw the mess up of the date here — not sure I want an 18-month year! Will fold that into the other review feedback.